docs/collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/recommended-tracking-design-web-mobile/index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Recommended Tracking Design for Web or Mobile Events"
+date: "2023-10-12"
+sidebar_position: 200
+---
+
+# Recommended Tracking Design for Web or Mobile Events
+
+Depending on your business needs you may decide to come up with your own custom context entities / self-describing events. However, for out-of-the-box solutions Snowplow designed a set of context entities specifically useful for web and mobile tracking which we used as a base for our data models, too. 
+
+The below table provides a list of our recommended web & mobile context entities and also shows how to enable them in your modeled data (the *snowplow-unified* package has all of them but the mobile and web only contexts are also available to use in their respective platform specific data models under the same variable names).
+
+:::tip
+Snowplow has started to move away from the wide, out-of-the-box canonical event structure and as part of this process some of these vital fields are now recommended to be tracked and used from purpose built topical context entities such as below. The snowplow-unified data model structure should enable this transition to be smooth: with a set of coalesce(..) statements you can change the source of a specific field in the `unify_fields_query()` macro, if necessary. You can check the full list of fields and from where we recommend taking them for both web and mobile [here](/docs/understanding-your-pipeline/canonical-event/index.md).
+:::
+
+| Context               | Web | Mobile | Recommendation                                                           | How to enable it in the trackers/pipeline | How to enable it in the unified model                      | Field name                                                  |
+|-----------------------|-----|--------|--------------------------------------------------------------------------|-------------------------------------------|------------------------------------------------------------|-------------------------------------------------------------|
+| page view context     | ✅   | ❌      | required for web (needed for page_view_id)                               | autotracked by default for web trackers   | snowplow__enable_web: true                                 | com_snowplowanalytics_snowplow_web_page_1                   |
+| screen view events    | ❌   | ✅      | required for mobile (needed for screen_view_id)                          | must be manually tracked                  | snowplow__enable_mobile: true                              | com_snowplowanalytics_mobile_screen_view_1                  |
```

Comment:
These are events that don't really fit under the Context table. Maybe we can create a separate table for OOTB events? That would also including background, foreground, install, deep link, link tracking and other events?